.github/workflows/release.yaml

```diff
@@ -52,7 +52,7 @@ jobs:
         if: github.ref_type == 'tag'
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: gh release upload ${{ github.ref_name }} prefect-crds.yaml prefect.yaml
+        run: gh release upload ${{ github.ref_name }} prefect-crds.yaml
```

Comment:
I missed this too - previously, `prefect-crds.yaml` had only the CRDs, and `prefect.yaml` had the CRDS _and_ the Deployment, etc, so folks could add it all in one. We could still populate this by running `helm template` and piping it out to a file, but I'm fine skipping this for now as I bet most will install it via helm.

Reply:
i'm happy to do that if you think thats useful!

Reply:
can just be a follow-up issue i think, i don't see it as a big priority unless users request it

Reply:
follow up issue [here](https://linear.app/prefect/issue/PLA-334/update-prefect-operator-to-package-manifests-alongside-release)